### PR TITLE
properly link `libjvm.so/dynlib`

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/openjdk any idea why osx is not behaving?

**Comment by CJ-Wright** [comment]
@isuruf I think this is ready for review (I wanted to check that this was what you intended)

File: recipe/build.sh [hunk 1]
```diff
@@ -93,9 +93,12 @@ if [[ "$target_platform" == linux* ]]; then
   # images where these fonts are missing (thanks to @chapmanb)
   mkdir -p $INSTALL_DIR/lib/fonts
   mv $SRC_DIR/fonts/ttf/* $INSTALL_DIR/lib/fonts/
+  # Have libjvm.so show up in the right place
+  ln -s $PREFIX/lib/server/libjvm.so $PREFIX/lib/libjvm.so
```
Comment (isuruf) [inline recipe/build.sh]:
```suggestion
  ln -s $INSTALL_DIR/lib/server/libjvm.so $INSTALL_DIR/lib/libjvm.so
```

Comment (CJ-Wright) [reply recipe/build.sh]: Thank you!

**Comment by CJ-Wright** [comment]
@conda-forge/openjdk @isuruf I think this is ready to go.

**Comment by CJ-Wright** [comment]
@conda-forge/core can I get some review/merge? This is holding up a few downstream recipes that I'd like to build.

**Comment by isuruf** [comment]
Looks good to me. I'll merge if you agree to become a maintainer.

**Comment by CJ-Wright** [comment]
Sure, not certain how much help I can provide (in terms of knowledge) but I'll see what I can do.

**Comment by CJ-Wright** [comment]
I guess I also need to backport this to the other versions.

**Comment by CJ-Wright** [comment]
Ah, and I need to bump the build number

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**